package/base-files/files/sbin/sysupgrade

```diff
@@ -161,35 +175,37 @@ add_overlayfiles() {
 
 		# backup files from /etc/sysupgrade.conf and /lib/upgrade/keep.d, but
 		# ignore those aready controlled by opkg conffiles
-		list_static_conffiles | sort -u |
-			grep -h -v -x -F -f $conffiles > "$keepfiles"
+		list_static_conffiles | sort -u \
+			| grep -h -v -x -F -f $conffiles > "$keepfiles"
```

Comment:
Single line.

Reply:
I don't want to exceed 88 chars per line.